utils/createQueryKeys.ts

```diff
@@ -0,0 +1,11 @@
+// ref: https://tkdodo.eu/blog/effective-react-query-keys
+export const createQueryKeys = <T extends string>(keyword: T) => {
+  const keys = {
+    all: [keyword] as const,
+    lists: () => [...keys.all, 'list'] as const,
+    list: (args: unknown[]) => [...keys.lists(), ...args] as const,
+    details: () => [...keys.all, 'detail'] as const,
+    detail: (args: unknown[]) => [...keys.details(), ...args] as const,
```

Comment:
👍🏻👍🏻

Reply:
@young-do @siyeons 혹시 아래와 같이 변경해도 되나요? 이유는 더 유연한 키 관리를 위해서 입니다!
```suggestion
    lists: () => [...keys.all, 'list'] as const,
    list: (args: any) => [...keys.lists(), args] as const,
    details: () => [...keys.all, 'detail'] as const,
    detail: (args: any) => [...keys.details(), args] as const,
```

Reply:
오 그런점은 생각 못 했네요! 👍 
반영하겠습니다 :)

Reply:
근데 생각해보니 실제 사용하는 곳에서
지금 방식으로 `SOME_KEY.list([ { eventId, groupId } ])` 처럼 사용하면 될 것 같기도 하네요!

키를 넘겨줄 때 기존에 배열로 입력 가능하던걸 그대로 유지하고 싶어서 타입을 `unknown[]`로 한거 거든요!

Reply:
아하 그러면 영도님이 말씀해주신대로 하겠슴다! 🙌